### ISSUE TITLE
Visual distinction for notes

### ISSUE THREAD

**Issue by dentonjacobs** [ronilaukkarinen/plaintasks-markdown#1]
PlainTasks supports notes within the task list. I love this theme, but would like to have a better visual indication of notes vs. tasks. Right now, they appear to be treated the same, so they have the same font, size, color, etc. A visual distinction would be great to distinguish the two when needed.

For example:
![sample-tasklist](https://cloud.githubusercontent.com/assets/1913818/7836488/312d706e-0446-11e5-92ca-2d65c060ccb8.PNG)

**Comment by rollecode** [comment]
Thanks for the input! I use notes in PlainTasks very rarely. What do you suggest? different color?

**Comment by dentonjacobs** [comment]
Yes, a different color could be awesome. For me, the ideal would be to have comments in the same gray as completed items. I basically always archive as soon as I mark complete, so gray would be perfect. Thanks for considering it!
